build/Package-Nuget.ps1

```diff
@@ -0,0 +1,45 @@
+# Copy files needed for Nuget package generation to appropriate directory and create Nuget package.
+# Then copy Nuget package over to bin\nupkg directory to be used by the Azure Devops pipeline.
+
+# Creating path variables for directories where files are located and their target locations.
+$RootPath = Split-Path $PSScriptRoot -Parent
+$SceneLoader = "$RootPath\SceneLoader"
+$NugetPackager = "$SceneLoader\NugetPackager\"
+$lib = "$NugetPackager\lib\uap10.0"
+$runtimesx64 = "$NugetPackager\runtimes\win10-x64\native\"
+$runtimesx86 = "$NugetPackager\runtimes\win10-x86\native\"
+$runtimesARM = "$NugetPackager\runtimes\win10-ARM\native\"
+
+$outputDirx64 = "$RootPath\x64\Release\SceneLoader"
+$outputDirx86 = "$RootPath\Release\SceneLoader"
+$outputDirARM = "$RootPath\ARM\Release\SceneLoader"
+$nupkg = "$RootPath\bin\nupkg"
+
+# Copy .idl and .winmd files over to $lib folder.
+Copy-Item -Path "$SceneLoader\SceneLoaderComponent.idl" -Destination $lib
```

Comment:
>idl [](start = 51, length = 3)

Do you definitely need the IDL file? It contains the same info as the .winmd file. I thought the C++ compiler for CX understood winmd, and so does the CPPWinRT tool.